azext_iot/central/params.py

```diff
@@ -38,7 +38,7 @@
 api_version = CLIArgumentType(
     options_list=["--api-version", "--av"],
```

Comment:
Wasn't a goal to deprecate and remove `--api-version` ? You could add the deprecation attribute in this change set, warning users of the impending removal.